tests/common/member_test.go

```diff
@@ -152,3 +152,100 @@ func TestMemberAdd(t *testing.T) {
 		}
 	}
 }
+
+func TestMemberRemove(t *testing.T) {
+	testRunner.BeforeTest(t)
+
+	tcs := []struct {
+		name                  string
+		strictReconfigCheck   bool
+		waitForQuorum         bool
+		expectSingleNodeError bool
+		expectClusterError    bool
+	}{
+		{
+			name:                  "StrictReconfigCheck/WaitForQuorum",
+			strictReconfigCheck:   true,
+			waitForQuorum:         true,
+			expectSingleNodeError: true,
+		},
+		{
+			name:                  "StrictReconfigCheck/NoWaitForQuorum",
+			strictReconfigCheck:   true,
+			waitForQuorum:         false,
+			expectSingleNodeError: true,
+			expectClusterError:    true,
+		},
+		{
+			name:                "DisableStrictReconfigCheck/WaitForQuorum",
+			strictReconfigCheck: false,
+			waitForQuorum:       true,
+		},
+		{
+			name:                "DisableStrictReconfigCheck/NoWaitForQuorum",
+			strictReconfigCheck: false,
+			waitForQuorum:       false,
```

Comment:
Sometimes you use the golang build-in zero value `false`, but sometimes you explicitly provide a `false`. I suggest to keep it consistent. I suggest to only provide `true` if you want to enable it, and no need to provide `false` for the flag which you don't want to enable, because they will have `false` by default

Reply:
Thanks! I'd like to keep it consistent.